### PR TITLE
Adapt devpack contract classes

### LINKED ISSUE #292: Add `getHash()` to all native contract classes in the devpack
The dotnet devpack has added getters for the contract hash of native contract classes. Neow3j devpack should follow this feature.

Proposal: Add a `getHash()` method that gets transformed into a `PUSHDATA xx <hash>`.

### PR DESCRIPTION
**Pull request by csmuller**
Closes #292, #298 and #299.

### CODE REVIEW

**Comment by csmuller** [comment]
Write integration tests for NeoToken, GasToken, and PolicyContract.

File: devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java [hunk 1]
```diff
@@ -97,5 +90,40 @@
      */
     public static native boolean isBlocked(byte[] scriptHash);
 
+    /**
+     * Gets the fee factor used to calculate the GAS cost of contract executions.
+     * <p>
+     * Each neo-vm instruction has a relative cost that is multiplied with this fee factor to arrive
```
Comment (gsmachado) [inline devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java:96]:
```suggestion
     * Each neo-vm instruction has a relative cost that is multiplied with this fee factor to result
```

File: devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java [hunk 2]
```diff
@@ -97,5 +90,40 @@
...
+     * at the actual GAS cost.
```
Comment (gsmachado) [inline devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java:97]:
```suggestion
     * in the actual GAS cost.
```

File: devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java [hunk 3]
```diff
@@ -97,5 +90,40 @@
...
+     *
+     * @return the execution fee factor.
+     */
+    public static native int getExecFeeFactor();
+
+    /**
+     * Sets the fee factor used to calculate the GAS cost of contract executions.
+     * <p>
+     * Each neo-vm instruction has a relative cost that is multiplied with this fee factor to arrive
```
Comment (gsmachado) [inline devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java:106]:
```suggestion
     * Each neo-vm instruction has a relative cost that is multiplied with this fee factor to result
```

File: devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java [hunk 4]
```diff
@@ -97,5 +90,40 @@
...
+     * at the actual GAS cost.
```
Comment (gsmachado) [inline devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java:107]:
```suggestion
     * in the actual GAS cost.
```

**Review by gsmachado — APPROVED** [review]
(super) minor changes. 💯

**Review by gsmachado — APPROVED** [review]

**Review by gsmachado — APPROVED** [review]
💯  👍  🍾 

to the moon! 🚀

**Pull request merged**